tests/conftest.py

```diff
@@ -1,22 +1 @@
-import pytest
-import os
-
-# Import the functional fixtures as a plugin
-# Note: fixtures with session scope need to be local
-
 pytest_plugins = ["dbt.tests.fixtures.project"]
-
-
-# The profile dictionary, used to write out profiles.yml
-@pytest.fixture(scope="class")
-def dbt_profile_target():
```

Comment:
This got moved within the `tests/functional` scope as live database credentials should never be needed for unit testing.

Reply:
make sure to run the unit test workflow manually to validate we don't have some weird dependency on it

Reply:
Agreed, but I'd almost rather fix that than move it back, or just move the tests into `functional`.

Reply:
These pass.